pkg/detectors/grafana/grafana.go

```diff
@@ -85,3 +75,41 @@ func (s Scanner) Type() detectorspb.DetectorType {
 func (s Scanner) Description() string {
 	return "Grafana is an open-source platform for monitoring and observability. Grafana API keys can be used to access and manage Grafana resources."
 }
+
+func verifyGrafanaKey(ctx context.Context, client *http.Client, token string) (bool, error) {
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, "https://grafana.com/api/v1/tokens?region=us", http.NoBody)
```

Comment:
Could there be any other regions? More specifically, can there be scenarios where one token might be for some region other than us?

Reply:
Grafana Tokens are not tied to any specific region.